roles/sensu/server/templates/sensu_oschecks.json.j2

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
The whole value of "checks" hash has to be configurable by operator. In other words, it will be operator who will define what checks and where will run.